jobs/build-charms/charms.py

```diff
@@ -405,9 +422,10 @@ def resource_spec(self):
         return self.db["build_args"].get("resource_spec", None)
 
     @property
-    def to_channel(self):
-        """Get destination channel."""
-        return self.db["build_args"].get("to_channel", None)
+    def to_channels(self):
+        """Get destination channels."""
+        chan = self.db["build_args"].get("to_channel", None)
+        return matched_numerical_channel(chan, SNAP_K8S_TRACK_MAP)
```

Comment:
so, it supports either single work "risk" or a full "channel"

`edge` is acceptable
`1.25/stable` is acceptable. 

there's quite a bit of alignment between this file and the [bundle script](https://github.com/charmed-kubernetes/bundle/blob/main/bundle) going on -- not to mention historically this has been called `channel` because it was a charmstore `channel` which aligns with a `charmhub risk` better.  I suppose we could go whole hog here removing ALL the publish to charmstore business and re-vamp this file to ONLY push to charmhub.